k8s.io/configmap-nginx.yaml

```diff
@@ -255,8 +255,9 @@ data:
         listen 443 ssl;
 
         location / {
-          rewrite ^/$                  https://github.com/kubernetes/kubernetes/releases redirect;
-          rewrite ^/([^/]*)(/.*)?$     https://github.com/kubernetes/kubernetes/tree/$1$2 redirect;
+          rewrite ^/$                       https://github.com/kubernetes/kubernetes/releases redirect;
+          rewrite ^/(master|HEAD)(/.*)?$    https://github.com/kubernetes/examples/tree/$1 redirect;
```

Comment:
should probably change the regex to include `examples` too, as otherwise something like https://releases.k8s.io/master/pkg/kubectl will incorrectly get redirected to the examples repo.

Reply:
We could PROBABLY just break examples here, if we need to.